### PR TITLE
feat(recipes): add baked beans recipe

### PR DESCRIPTION
**Pull request by alir**
beans means heinz

### CODE REVIEW

File: recipes/baked_beans.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "recipe_name": "Baked Beans",
+    "serves": "1",
```
Comment (Goregius) [inline recipes/baked_beans.json:3]: This is a bit sad, maybe make it 2 and include more beans?

File: recipes/baked_beans.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "prep_time": "2 mins",
```
Comment (Goregius) [inline recipes/baked_beans.json:4]: A strong guy like me would only take 1 minute to open a can of baked beans so lets make it 1 min.

File: recipes/baked_beans.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "cooking_time": "2 mins",
```
Comment (Goregius) [inline recipes/baked_beans.json:5]: Heinz says to cook it for 2.5 minutes and I can't afford a powerful microwave so lets make it 3 minutes.

Reference: https://www.kff.co.uk/heinz-baked-beans/p/11530

File: recipes/baked_beans.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "ingredients": [
+        {
+            "unit_size": "1",
+            "unit_type": "tin",
+            "ingredient_name": "heinz baked beans"
```
Comment (Goregius) [inline recipes/baked_beans.json:10]:
```suggestion
            "ingredient_name": "Branston baked beans"
```
Branston beans are far superior to Heinz beans. The thick sauce and much more developed flavour really sets it apart. 

Source: https://www.dailymail.co.uk/news/article-4548674/Heinz-left-trailing-place-blind-taste-test.html

File: recipes/baked_beans.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        }
+    ],
+    "method": [
+        "Empty tin of baked beans into a bowl.",
```
Comment (Goregius) [inline recipes/baked_beans.json:14]:
```suggestion
        "Empty tin of baked beans into a bowl, cover and vent.",
```

True microwave connoisseurs cover and vent their beans.

**Review by Goregius — CHANGES_REQUESTED** [review]
Good initial recipe, but choosing Heinz over Branston beans really makes me question your life decisions.

File: recipes/baked_beans.json (hunk 1 above)
Comment (alir) [reply recipes/baked_beans.json:3]: Two tins of beans for two people?

Comment (alir) [reply recipes/baked_beans.json]: Done

File: recipes/baked_beans.json (hunk 3 above)
Comment (alir) [reply recipes/baked_beans.json]: Sounds good. Adjusted cooking and prep time

**Comment by gowhale** [comment]
Nice one @alir! I don't like beans but merge from main to run the GitHub actions I just added then I will merge! 💪

**Comment by gowhale** [comment]
Thanks for being the first contributor to test the GitHub workflows @alir ! 🙇‍♂️

**Pull request merged**